cmd/queue/main.go

```diff
@@ -201,7 +201,7 @@ func main() {
 		// when we're actually in the same container.
 		transport := &http.Transport{
 			DialContext: func(_ context.Context, _, _ string) (net.Conn, error) {
-				return net.Dial("unix", unixSocketPath)
+				return net.Dial("unix", os.TempDir()+unixSocketPath)
```

Comment:
nit, but better to do `filepath.Join` here I think

Reply:
Thank you. Fixed.